rest/hosts_pool.go

```diff
@@ -192,17 +191,18 @@ func (s *Server) getHostInPool(w http.ResponseWriter, r *http.Request) {
 
 func (s *Server) listHostsInPool(w http.ResponseWriter, r *http.Request) {
 	filtersString := r.URL.Query()["filter"]
+
 	filters := make([]labelsutil.Filter, len(filtersString))
 	for i := range filtersString {
 		var err error
-		str, err := url.QueryUnescape(filtersString[i])
-		if err != nil {
-			writeError(w, r, newBadRequestError(fmt.Errorf("%v with filter %s", err, filtersString)))
-			return
-		}
-		filters[i], err = labelsutil.CreateFilter(str)
+		/*	str, err := url.QueryUnescape(filtersString[i])
```

Comment:
Just remove this instead of commenting it

Reply:
Oh, my bad !